lib/utils.ts

```
@@ -1,7 +1,21 @@
-import type { CoreAssistantMessage, CoreToolMessage, UIMessage } from 'ai';
-import { type ClassValue, clsx } from 'clsx';
-import { twMerge } from 'tailwind-merge';
 import type { DBMessage, Document } from '@/lib/db/schema';
+import { createOpenRouter } from '@openrouter/ai-sdk-provider';
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)